### PR TITLE
pipenv: add page

### PR DESCRIPTION
**Pull request by heimann**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/pipenv.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# pipenv
+
+> Python Development Workflow for Humans.
```
Comment (sbrl) [inline pages/common/pipenv.md]: Perhaps this would read better in sentence case rather than title case?

File: pages/common/pipenv.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> Manages packages and virtual environment for a project.
```
Comment (sbrl) [inline pages/common/pipenv.md]: Perhaps `the virtual environment` would read better here?

File: pages/common/pipenv.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Create a new project using Python 3:
+
+`pipenv --three`
+
+- Install a package:
+
+`pipenv install {{package_name}}`
+
+- Install all dependencies for a project (including dev):
```
Comment (sbrl) [inline pages/common/pipenv.md]: perhaps `all the dependencies` would read better here?

File: pages/common/pipenv.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`pipenv install --dev`
+
+- Uninstall a package:
+
+`pipenv uninstall {{package_name}}`
+
+- Activate a shell within the created virtual environment:
```
Comment (sbrl) [inline pages/common/pipenv.md]: Perhaps `Start` would aid clarity here?

Comment (heimann) [reply pages/common/pipenv.md]: `Start a shell within...` etc?

File: pages/common/pipenv.md (hunk 1 above)
Comment (heimann) [reply pages/common/pipenv.md]: I agree

File: pages/common/pipenv.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# pipenv
+
+> Python development workflow for humans.
```
Comment (agnivade) [inline pages/common/pipenv.md]: The "for humans" part makes it a bit informal. Can we make it "Simple and unified Python packaging tool" ?

Comment (heimann) [reply pages/common/pipenv.md]: How about "Python packaging and virtual environments made simple."? Key part of the program is that it handles not just package management but the whole environment around it.

Another option could be: "Simple and unified Python development workflow"?

Comment (agnivade) [reply pages/common/pipenv.md]: I like the second one.

Comment (heimann) [reply pages/common/pipenv.md]: Ok done. Thanks for the feedback!

**Pull request merged**